package.json

```diff
@@ -41,47 +42,47 @@
     "@types/rsvp": "^4.0.3",
     "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-auto-import": "^2.4.0",
-    "ember-cli": "~3.28.2",
-    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli": "~4.8.0",
+    "ember-cli-dependency-checker": "^3.3.1",
     "ember-cli-deprecation-workflow": "^2.1.0",
-    "ember-cli-fastboot": "^3.3.0",
+    "ember-cli-fastboot": "^4.1.0",
     "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-memory-leak-detector": "^0.5.1",
-    "ember-cli-sass": "^10.0.1",
+    "ember-cli-memory-leak-detector": "0.5.1",
+    "ember-cli-sass": "^11.0.1",
     "ember-cli-terser": "^4.0.2",
     "ember-cli-typescript-blueprints": "^3.0.0",
-    "ember-code-example-component": "^0.3.1",
-    "ember-code-snippet": "2.3.1",
-    "ember-concurrency": "^2.1.0",
+    "ember-code-example-component": "^0.4.1",
+    "ember-code-snippet": "git+https://github.com/ef4/ember-code-snippet.git#d054b697098ad52481c94a952ccf8d89ba1f25fe",
```

Comment:
This seems to break CI. It seems to be translated to SSH protocol somehow and than failing with a SSH key error.

```
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/ef4/ember-code-snippet.git
npm ERR! 
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR! 
npm ERR! exited with error code: 128
```

Reply:
exakt, do you have any idea, how i can fix it?
locally it works without any problems

Reply:
It seems to be caused by `resolved` entry in `package-lock.json`:

```json
"node_modules/ember-code-snippet": {
      "version": "3.0.0",
      "resolved": "git+ssh://git@github.com/ef4/ember-code-snippet.git#d054b697098ad52481c94a952ccf8d89ba1f25fe",
      "integrity": "sha512-9YK+QWva4TqCKkyRpjjAHxf/qB+Jr089ElEaGcLO2fgbg+lLhCwMrljKEAutONVHTHuoQ87Qd/de5g7Hrk2X5Q==",
      "dev": true,
      "license": "MIT",
      "dependencies": {
        "broccoli-flatiron": "^0.1.3",
        "broccoli-merge-trees": "^1.0.0",
        "broccoli-plugin": "^1.3.1",
        "ember-cli-babel": "^7.7.3",
        "ember-cli-htmlbars": "^3.0.1 || ^4.0.0 || ^5.0.0 || ^6.0.0",
        "es6-promise": "^1.0.0",
        "glob": "^7.1.3"
      },
      "engines": {
        "node": "8.* || >= 10.*"
      }
    },
```

@mkszepp Maybe using `ssh` instead of `https` protocol is caused by your local setup?

Reply:
@jelhan tried, but has not fixed... :(

Reply:
@jelhan found a fix... https://github.com/actions/setup-node/issues/214#issuecomment-810829250